node/api/api.go

```diff
@@ -162,3 +184,77 @@ func WriteJSON(w http.ResponseWriter, obj interface{}) {
 func WriteSuccess(w http.ResponseWriter) {
 	w.WriteHeader(http.StatusNoContent)
 }
+
+func GetPaginationDefaultRequest(req *http.Request) PaginationRequest {
+	return GetPaginationRequest(req, "start", "limit")
```

Comment:
I'm not sure about the custom names for `start` and `limit`. If we do that, the names of the fields of `PaginationResponse` should probably also be custom to match the fields in the request.